rental.rb

```diff
@@ -1,13 +1,13 @@
+require_relative 'book'
 class Rental
-  attr_reader :book, :person
-  attr_accessor :date
+  attr_accessor :date, :book, :person
 
   def initialize(date, book, person)
     @date = date
     @book = book
-    book.rentals << self
+    book.rental << self
```

Comment:
- [ ] Your app is breaking in attempt to create a rental for a person. Just after a date is entered for rental the app crashes. Kindly ensure to improve upon this aspect.

**Creating rental error:**
![image](https://user-images.githubusercontent.com/12745474/183095047-d48de7d2-ff86-4e8c-88b5-edddfa59fb96.png)